content/docs/get-started/index.md

```diff
@@ -59,43 +63,39 @@ save(
 )
 ```
 
-Here, we loaded a well-known
-[Iris flower dataset](https://archive.ics.uci.edu/ml/datasets/iris) with
-`scikit-learn` and trained a simple classifier. Instead of pickling the model,
-we persisted it using MLEM's `save` API.
+Here, we loaded a well-known [Iris flower dataset] with scikit-learn and trained
+a simple classifier (see the full list of [supported ML frameworks]). We then
+captures it using `mlem.api.save`, which creates the following files:
```

Comment:
```suggestion
capture it using `mlem.api.save`, which creates the following files:
```

Reply:
Why not "persist"? "capture" sounds strange to me (may be just my bad English 😅 )

Reply:
"to persist" doesn't mean what we want it to 🙂 

See discussion in https://github.com/iterative/dvc.org/issues/2960